topics/server.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Valkey Server"
+linkTitle: "Valkey Server"
+weight: 1
+description: >
+    Manual for valkey-server, the Valkey server program
+---
+
+## Usage
+
+**`valkey-server`** [ _/path/to/valkey.conf_ ] [ _OPTIONS_ ] [**`-`**]\
+**`valkey-server`** **`-`**\
+**`valkey-server`** **`-v`** | **`--version`**\
+**`valkey-server`** **`-h`** | **`--help`**\
+**`valkey-server`** **`--test-memory`** _megabytes_\
+**`valkey-server`** **`--check-system`**
+
+## Description
+
+`valkey-server` is the Valkey database program. It's typically started by an init script, rather than manually.
+
+## Options
+
+The configuration file and the configuration directives are in documented in
+[Configuration](valkey.conf.md). Use `-` to read configuration from stdin.
+
+Each of the configuration directives can be provided on the command line
+with its name prefixed by two dashes. For example, `--port 6380` on the command
+line is equivalent to `port 6380` in the config file.
+
+Additional options:
+
+**`-v`**, **`--version`**
+: Output version and exit.
+
+**`-h`**, **`--help`**
+: Output help and exit.
+
+**`--test-memory`** _megabytes_
+: Run a memory test and exit.
+
+**`--check-sytem`**
+: Output some operating system properties relevant for running Valkey and exit.
+
+**`--sentinel`**
+: Start in [sentinel](sentinel.md) mode
+
+## Examples
+
+Run the server with default config:
+
+    valkey-server
+
+Read configuration from stdin:
+
+    echo 'maxmemory 128mb' | valkey-server -
+
+Start with a configuration file:
+
+    valkey-server /etc/valkey/6379.conf
+
+Start with configuration as command line options:
+
+    valkey-server --port 7777
+
+Start as a replica of another node running on the same machine:
+
+    valkey-server --port 7777 --replicaof 127.0.0.1 8888
+
+Start with a config file, then some additional options overriding the ones in
+the config file, and finally some more options from stdin:
+
+    valkey-server /etc/myvalkey.conf --loglevel verbose -
+
+Start with a config file and some additional options overriding the ones in
+the config file:
+
+    valkey-server /etc/myvalkey.conf --loglevel verbose
+
+Start in [sentinel](sentinel.md) mode:
+
+    valkey-server /etc/sentinel.conf --sentinel
+
```

Comment:
```suggestion
```
Is this relevant here? It's not listed as one of the configuration options above.

Reply:
It is listed on line 45 (under Options).

Again this is from `valkey-server --help`. Anyway, I'll take your suggestion.

Reply:
It isn't a configuration option. It's listed under "Additional options".

Reply:
Ok, so if it's considered an option, why is it the only listed option?

Reply:
It's also not really an option, because you can't pass it in the config file, it has to be a startup argument.